github.md

```diff
@@ -10,23 +18,33 @@ Branch protection should be enabled for the default/release branch of a reposito
 - Require branches to be up to date before merging
 - Include administrators
 
-## Codeowners
+### Access
+- Access should be granted to [GitHub teams][gh-teams]
+  - Should an individual leave the GitHub organisation, they'll automatically lose access to any repository in the organisation
+- Avoid individual access
+  - Should the individual leave the GitHub organisation, they'll retain access until manually removed
+- Open the repository to the department
+  - [`@guardian/guardian-developers-read`][gh-read] should have read access
+  - [`@guardian/guardian-developers-write`][gh-write] should have write access
```

Comment:
I agree with this, _unless_ one of the later guidelines isn't followed (committing secrets to a private repo). In that scenario, limiting access is warranted, but obviously we should be fixing the actual issue.

Only mentioning because I saw an example recently.

Reply:
Hmm, as these are recommendations, I think this scenario is implicitly covered? (Wait, are implicits good?!)